garage/tf/models/gaussian_mlp_model.py

```diff
@@ -0,0 +1,227 @@
+"""Gaussian MLP Model."""
+import numpy as np
+import tensorflow as tf
+
+from garage.core import Serializable
+from garage.misc import ext
+from garage.misc.overrides import overrides
+from garage.tf.core.mlp import mlp
+from garage.tf.core.parameter import parameter
+from garage.tf.models import Model
+
+
+class GaussianMLPModel(Model, Serializable):
+    """
+    Gaussian MLP Model.
+
+    This model models a gaussian distribution whose
+    mean and std are parameterized by an MLP.
+    """
+
+    def __init__(
+            self,
+            input_dim,
+            output_dim,
+            name="GaussianMLPModel",
+            hidden_sizes=(32, 32),
+            learn_std=True,
+            init_std=1.0,
+            adaptive_std=False,
+            std_share_network=False,
+            std_hidden_sizes=(32, 32),
+            min_std=1e-6,
+            max_std=None,
+            std_hidden_nonlinearity=tf.nn.tanh,
+            hidden_nonlinearity=tf.nn.tanh,
+            output_nonlinearity=None,
+            std_parameterization='exp',
+    ):
+        """
```

Comment:
We are now putting constructor params in the class docstring rather than` __init__` docstring